### DISCUSSION TITLE
Retroachievements app

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1465] [Ideas]
Hi everyone!

Retroachievements tracking is one of the feature I love and use the most on Onion OS, but as off now there is no user-friendly way to navigate among the trophies.
The retroachievements API is stable and well documented.
Is there any project to make a standalone app to browse through trophies ? With network enabled and retroarch achievement settings it should not be very complicated, and getting progress data with wonderful cheevos icons would be a very nice to have.

Activity Tracker (UI) app would be a good start as a template.
If needed I can also help for the dev part

Any chance to have this app available soon?

**Comment by FDrebin** [comment]
A retroacheivements app would add an easy front end GUI to provide your credentials for.

**Reply by oherau** [reply]
You are totally right ! I had not that feature in mind but this new app could also be the main entry point to setup retroachievement credentials, not only for Retroarch but also for other standalone emulators using this feature !

**Comment by Nighted** [comment]
+1
